routes/words.js

```diff
@@ -2,13 +2,44 @@ const express = require('express')
 const router = express.Router()
 const checkAuthentificated = require('../public/checkAuthentificated').checkAuthentificated
```

Comment:
There's a typo in here, too: should be `authenticated`.

Jetbrains IDEs are awesome at this, as they have a spellchecker included. They've caught me quite a few times. But there's one for VS Code as well: see [this one](https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker), or [this one](https://marketplace.visualstudio.com/items?itemName=ban.spellright). I use this feature in Git GUIs as well, so I don't add typos in my commit messages either.

Reply:
I definitely need it because I struggle and doubt myself so much with some words

Reply:
Yeah, same here.
But I've two friends: 
- spellcheckers everywhere, and 
- doing `Ctrl + T` -> `define <word>` in gugal -> `Ctrl + W` .

Reply:
I have just read it out loud and I am laughing so hard 🤣 It sounds so bad